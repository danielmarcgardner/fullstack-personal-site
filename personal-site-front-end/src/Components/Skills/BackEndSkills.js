import React from 'react';
import './Skills.css';

const BackEndSkills = () => (
  <div className="skillscategory">
    <h1 className="skillsheader"> Back End Skills </h1>
    <div className="skilldiv">
      <img className="skillimage" alt="Node.js" src={require('../../Images/node.png')} />
    </div>
    <div className="skilldiv">
      <img className="skillimage" alt="Knex.js" src={require('../../Images/knex.png')} />
    </div>
    <div className="skilldiv">
      <img className="skillimage" alt="Bookshelf.js" src={require('../../Images/bookshelf.png')} />
    </div>
    <div className="skilldiv">
      <img className="skillimage" alt="PostgreSQL" src={require('../../Images/postgresql.png')} />
    </div>
    <div className="skilldiv">
      <img className="skillimage" alt="Swagger.js" src={require('../../Images/swagger.png')} />
    </div>
    <div className="skilldiv">
      <img className="skillimage" alt="Json Web Token" src={require('../../Images/jwt.png')} />
    </div>
    <div className="skilldiv">
      <img className="skillimage" alt="Oauth" src={require('../../Images/oauth.png')} />
    </div>
    <div className="skilldiv">
      <img className="skillimage" alt="Express.js" src={require('../../Images/express.png')} />
    </div>
  </div>
);

export default BackEndSkills;
