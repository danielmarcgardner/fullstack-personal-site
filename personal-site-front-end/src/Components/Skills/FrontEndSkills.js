import React from 'react';
import './Skills.css';

const FrontEndSkills = () => (
  <div className="skillscategory">
    <h1 className="skillsheader"> Front End Skills </h1>
    <div className="skilldiv">
      <img className="skillimage" alt="JavaScript" src={require('../../Images/js.png')} />
    </div>
    <div className="skilldiv">
      <img className="skillimage" alt="React" src={require('../../Images/react.png')} />
    </div>
    <div className="skilldiv">
      <img className="skillimage" alt="Redux" src={require('../../Images/redux.png')} />
    </div>
    <div className="skilldiv">
      <img className="skillimage" alt="HTML5" src={require('../../Images/html5.png')} />
    </div>
    <div className="skilldiv">
      <img className="skillimage" alt="CSS3" src={require('../../Images/css3.png')} />
    </div>
    <div className="skilldiv">
      <img className="skillimage" alt="React Router" src={require('../../Images/reactrouter.png')} />
    </div>
    <div className="skilldiv">
      <img className="skillimage" alt="Semantic UI" src={require('../../Images/semantic.png')} />
    </div>
    <div className="skilldiv">
      <img className="skillimage" alt="Materialize" src={require('../../Images/materialize.png')} />
    </div>
  </div>
);

export default FrontEndSkills;
