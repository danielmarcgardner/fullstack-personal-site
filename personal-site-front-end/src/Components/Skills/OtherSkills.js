import React from 'react';
import './Skills.css';

const OtherSkills = () => (
  <div className="skillscategory">
    <h1 className="skillsheader"> Testing & Other Skills </h1>
    <div className="skilldiv">
      <img className="skillimage" alt="Jest" src={require('../../Images/jest.png')} />
    </div>
    <div className="skilldiv">
      <img className="skillimage" alt="Mocha" src={require('../../Images/mocha.png')} />
    </div>
    <div className="skilldiv">
      <img className="skillimage" alt="Chai" src={require('../../Images/chai.png')} />
    </div>
    <div className="skilldiv">
      <img className="skillimage" alt="Super Test" src={require('../../Images/supertest.png')} />
    </div>
    <div className="skilldiv">
      <img className="skillimage" alt="Git" src={require('../../Images/git.png')} />
    </div>
    <div className="skilldiv">
      <img className="skillimage" alt="Github" src={require('../../Images/github.png')} />
    </div>
    <div className="skilldiv">
      <img className="skillimage" alt="Heroku" src={require('../../Images/heroku.png')} />
    </div>
    <div className="skilldiv">
      <img className="skillimage" alt="NPM" src={require('../../Images/npm.png')} />
    </div>
  </div>
);

export default OtherSkills;
