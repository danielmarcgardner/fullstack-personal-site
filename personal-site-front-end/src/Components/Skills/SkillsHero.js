import React from 'react';
import { Grid, Container } from 'semantic-ui-react';
import FrontEndSkills from './FrontEndSkills';
import BackEndSkills from './BackEndSkills';
import OtherSkills from './OtherSkills';
import './Skills.css';

const SkillsHero = () => (
  <div id="skillshero">
    <Container>
      <h1 id="skillsmainheader">Skills </h1>
      <Grid columns={3}>
        <Grid.Row>
          <Grid.Column>
            <FrontEndSkills />
          </Grid.Column>
          <Grid.Column>
            <BackEndSkills />
          </Grid.Column>
          <Grid.Column>
            <OtherSkills />
          </Grid.Column>
        </Grid.Row>
      </Grid>
    </Container>
  </div>
);

export default SkillsHero;
