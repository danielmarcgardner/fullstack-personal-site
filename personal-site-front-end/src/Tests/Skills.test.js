import React from 'react';
import { shallow } from 'enzyme';
import SkillsHero from '../Components/Skills/SkillsHero';
import FrontEndSkills from '../Components/Skills/FrontEndSkills';
import BackEndSkills from '../Components/Skills/BackEndSkills';
import OtherSkills from '../Components/Skills/OtherSkills';

const skills = shallow(<SkillsHero />);
const frontskills = shallow(<FrontEndSkills />);
const backskills = shallow(<BackEndSkills />);
const otherskills = shallow(<OtherSkills />);

describe('Skills', () => {
  it('SkillsHero renders', () => {
    expect(skills).toMatchSnapshot();
  });
  it('FrontEndSkills renders', () => {
    expect(frontskills).toMatchSnapshot();
  });
  it('BackEndSkills renders', () => {
    expect(backskills).toMatchSnapshot();
  });
  it('OtherSkills renders', () => {
    expect(otherskills).toMatchSnapshot();
  });
  it('SkillsHero has an id skillsmainheader', () => {
    expect(skills.find('#skillsmainheader').length).toEqual(1);
  });
  it('FrontEndSkills has 8 skilldivs and skill images', () => {
    expect(frontskills.find('.skilldiv').length).toEqual(8);
    expect(frontskills.find('.skillimage').length).toEqual(8);
  });
  it('BackEndSkills has 8 skilldivs and skill images', () => {
    expect(backskills.find('.skilldiv').length).toEqual(8);
    expect(backskills.find('.skillimage').length).toEqual(8);
  });
  it('OtherSkills has 8 skilldivs and skill images', () => {
    expect(otherskills.find('.skilldiv').length).toEqual(8);
    expect(otherskills.find('.skillimage').length).toEqual(8);
  });
});
