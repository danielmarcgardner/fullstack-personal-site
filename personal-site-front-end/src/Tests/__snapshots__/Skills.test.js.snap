// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Skills BackEndSkills renders 1`] = `
ShallowWrapper {
  "complexSelector": ComplexSelector {
    "buildPredicate": [Function],
    "childrenOfNode": [Function],
    "findWhereUnwrapped": [Function],
  },
  "length": 1,
  "node": <div
    className="skillscategory"
>
    <h1
        className="skillsheader"
    >
         Back End Skills 
    </h1>
    <div
        className="skilldiv"
    >
        <img
            className="skillimage"
            src="node.png"
        />
    </div>
    <div
        className="skilldiv"
    >
        <img
            className="skillimage"
            src="knex.png"
        />
    </div>
    <div
        className="skilldiv"
    >
        <img
            className="skillimage"
            src="bookshelf.png"
        />
    </div>
    <div
        className="skilldiv"
    >
        <img
            className="skillimage"
            src="postgresql.png"
        />
    </div>
    <div
        className="skilldiv"
    >
        <img
            className="skillimage"
            src="swagger.png"
        />
    </div>
    <div
        className="skilldiv"
    >
        <img
            className="skillimage"
            src="jwt.png"
        />
    </div>
    <div
        className="skilldiv"
    >
        <img
            className="skillimage"
            src="oauth.png"
        />
    </div>
    <div
        className="skilldiv"
    >
        <img
            className="skillimage"
            src="express.png"
        />
    </div>
</div>,
  "nodes": Array [
    <div
      className="skillscategory"
>
      <h1
            className="skillsheader"
      >
             Back End Skills 
      </h1>
      <div
            className="skilldiv"
      >
            <img
                  className="skillimage"
                  src="node.png"
            />
      </div>
      <div
            className="skilldiv"
      >
            <img
                  className="skillimage"
                  src="knex.png"
            />
      </div>
      <div
            className="skilldiv"
      >
            <img
                  className="skillimage"
                  src="bookshelf.png"
            />
      </div>
      <div
            className="skilldiv"
      >
            <img
                  className="skillimage"
                  src="postgresql.png"
            />
      </div>
      <div
            className="skilldiv"
      >
            <img
                  className="skillimage"
                  src="swagger.png"
            />
      </div>
      <div
            className="skilldiv"
      >
            <img
                  className="skillimage"
                  src="jwt.png"
            />
      </div>
      <div
            className="skilldiv"
      >
            <img
                  className="skillimage"
                  src="oauth.png"
            />
      </div>
      <div
            className="skilldiv"
      >
            <img
                  className="skillimage"
                  src="express.png"
            />
      </div>
</div>,
  ],
  "options": Object {},
  "renderer": ReactShallowRenderer {
    "_instance": ShallowComponentWrapper {
      "_calledComponentWillUnmount": false,
      "_compositeType": 2,
      "_context": Object {},
      "_currentElement": <BackEndSkills />,
      "_debugID": 5,
      "_hostContainerInfo": null,
      "_hostParent": null,
      "_instance": StatelessComponent {
        "_reactInternalInstance": [Circular],
        "context": Object {},
        "props": Object {},
        "refs": Object {},
        "state": null,
        "updater": Object {
          "enqueueCallback": [Function],
          "enqueueCallbackInternal": [Function],
          "enqueueElementInternal": [Function],
          "enqueueForceUpdate": [Function],
          "enqueueReplaceState": [Function],
          "enqueueSetState": [Function],
          "isMounted": [Function],
          "validateCallback": [Function],
        },
      },
      "_mountOrder": 3,
      "_pendingCallbacks": null,
      "_pendingElement": null,
      "_pendingForceUpdate": false,
      "_pendingReplaceState": false,
      "_pendingStateQueue": null,
      "_renderedComponent": NoopInternalComponent {
        "_currentElement": <div
          className="skillscategory"
>
          <h1
                    className="skillsheader"
          >
                     Back End Skills 
          </h1>
          <div
                    className="skilldiv"
          >
                    <img
                              className="skillimage"
                              src="node.png"
                    />
          </div>
          <div
                    className="skilldiv"
          >
                    <img
                              className="skillimage"
                              src="knex.png"
                    />
          </div>
          <div
                    className="skilldiv"
          >
                    <img
                              className="skillimage"
                              src="bookshelf.png"
                    />
          </div>
          <div
                    className="skilldiv"
          >
                    <img
                              className="skillimage"
                              src="postgresql.png"
                    />
          </div>
          <div
                    className="skilldiv"
          >
                    <img
                              className="skillimage"
                              src="swagger.png"
                    />
          </div>
          <div
                    className="skilldiv"
          >
                    <img
                              className="skillimage"
                              src="jwt.png"
                    />
          </div>
          <div
                    className="skilldiv"
          >
                    <img
                              className="skillimage"
                              src="oauth.png"
                    />
          </div>
          <div
                    className="skilldiv"
          >
                    <img
                              className="skillimage"
                              src="express.png"
                    />
          </div>
</div>,
        "_debugID": 6,
        "_renderedOutput": <div
          className="skillscategory"
>
          <h1
                    className="skillsheader"
          >
                     Back End Skills 
          </h1>
          <div
                    className="skilldiv"
          >
                    <img
                              className="skillimage"
                              src="node.png"
                    />
          </div>
          <div
                    className="skilldiv"
          >
                    <img
                              className="skillimage"
                              src="knex.png"
                    />
          </div>
          <div
                    className="skilldiv"
          >
                    <img
                              className="skillimage"
                              src="bookshelf.png"
                    />
          </div>
          <div
                    className="skilldiv"
          >
                    <img
                              className="skillimage"
                              src="postgresql.png"
                    />
          </div>
          <div
                    className="skilldiv"
          >
                    <img
                              className="skillimage"
                              src="swagger.png"
                    />
          </div>
          <div
                    className="skilldiv"
          >
                    <img
                              className="skillimage"
                              src="jwt.png"
                    />
          </div>
          <div
                    className="skilldiv"
          >
                    <img
                              className="skillimage"
                              src="oauth.png"
                    />
          </div>
          <div
                    className="skilldiv"
          >
                    <img
                              className="skillimage"
                              src="express.png"
                    />
          </div>
</div>,
      },
      "_renderedNodeType": 0,
      "_rootNodeID": 0,
      "_topLevelWrapper": null,
      "_updateBatchNumber": null,
      "_warnedAboutRefsInRender": false,
    },
    "getRenderOutput": [Function],
    "render": [Function],
  },
  "root": [Circular],
  "unrendered": <BackEndSkills />,
}
`;

exports[`Skills FrontEndSkills renders 1`] = `
ShallowWrapper {
  "complexSelector": ComplexSelector {
    "buildPredicate": [Function],
    "childrenOfNode": [Function],
    "findWhereUnwrapped": [Function],
  },
  "length": 1,
  "node": <div
    className="skillscategory"
>
    <h1
        className="skillsheader"
    >
         Front End Skills 
    </h1>
    <div
        className="skilldiv"
    >
        <img
            className="skillimage"
            src="js.png"
        />
    </div>
    <div
        className="skilldiv"
    >
        <img
            className="skillimage"
            src="react.png"
        />
    </div>
    <div
        className="skilldiv"
    >
        <img
            className="skillimage"
            src="redux.png"
        />
    </div>
    <div
        className="skilldiv"
    >
        <img
            className="skillimage"
            src="html5.png"
        />
    </div>
    <div
        className="skilldiv"
    >
        <img
            className="skillimage"
            src="css3.png"
        />
    </div>
    <div
        className="skilldiv"
    >
        <img
            className="skillimage"
            src="reactrouter.png"
        />
    </div>
    <div
        className="skilldiv"
    >
        <img
            className="skillimage"
            src="semantic.png"
        />
    </div>
    <div
        className="skilldiv"
    >
        <img
            className="skillimage"
            src="materialize.png"
        />
    </div>
</div>,
  "nodes": Array [
    <div
      className="skillscategory"
>
      <h1
            className="skillsheader"
      >
             Front End Skills 
      </h1>
      <div
            className="skilldiv"
      >
            <img
                  className="skillimage"
                  src="js.png"
            />
      </div>
      <div
            className="skilldiv"
      >
            <img
                  className="skillimage"
                  src="react.png"
            />
      </div>
      <div
            className="skilldiv"
      >
            <img
                  className="skillimage"
                  src="redux.png"
            />
      </div>
      <div
            className="skilldiv"
      >
            <img
                  className="skillimage"
                  src="html5.png"
            />
      </div>
      <div
            className="skilldiv"
      >
            <img
                  className="skillimage"
                  src="css3.png"
            />
      </div>
      <div
            className="skilldiv"
      >
            <img
                  className="skillimage"
                  src="reactrouter.png"
            />
      </div>
      <div
            className="skilldiv"
      >
            <img
                  className="skillimage"
                  src="semantic.png"
            />
      </div>
      <div
            className="skilldiv"
      >
            <img
                  className="skillimage"
                  src="materialize.png"
            />
      </div>
</div>,
  ],
  "options": Object {},
  "renderer": ReactShallowRenderer {
    "_instance": ShallowComponentWrapper {
      "_calledComponentWillUnmount": false,
      "_compositeType": 2,
      "_context": Object {},
      "_currentElement": <FrontEndSkills />,
      "_debugID": 3,
      "_hostContainerInfo": null,
      "_hostParent": null,
      "_instance": StatelessComponent {
        "_reactInternalInstance": [Circular],
        "context": Object {},
        "props": Object {},
        "refs": Object {},
        "state": null,
        "updater": Object {
          "enqueueCallback": [Function],
          "enqueueCallbackInternal": [Function],
          "enqueueElementInternal": [Function],
          "enqueueForceUpdate": [Function],
          "enqueueReplaceState": [Function],
          "enqueueSetState": [Function],
          "isMounted": [Function],
          "validateCallback": [Function],
        },
      },
      "_mountOrder": 2,
      "_pendingCallbacks": null,
      "_pendingElement": null,
      "_pendingForceUpdate": false,
      "_pendingReplaceState": false,
      "_pendingStateQueue": null,
      "_renderedComponent": NoopInternalComponent {
        "_currentElement": <div
          className="skillscategory"
>
          <h1
                    className="skillsheader"
          >
                     Front End Skills 
          </h1>
          <div
                    className="skilldiv"
          >
                    <img
                              className="skillimage"
                              src="js.png"
                    />
          </div>
          <div
                    className="skilldiv"
          >
                    <img
                              className="skillimage"
                              src="react.png"
                    />
          </div>
          <div
                    className="skilldiv"
          >
                    <img
                              className="skillimage"
                              src="redux.png"
                    />
          </div>
          <div
                    className="skilldiv"
          >
                    <img
                              className="skillimage"
                              src="html5.png"
                    />
          </div>
          <div
                    className="skilldiv"
          >
                    <img
                              className="skillimage"
                              src="css3.png"
                    />
          </div>
          <div
                    className="skilldiv"
          >
                    <img
                              className="skillimage"
                              src="reactrouter.png"
                    />
          </div>
          <div
                    className="skilldiv"
          >
                    <img
                              className="skillimage"
                              src="semantic.png"
                    />
          </div>
          <div
                    className="skilldiv"
          >
                    <img
                              className="skillimage"
                              src="materialize.png"
                    />
          </div>
</div>,
        "_debugID": 4,
        "_renderedOutput": <div
          className="skillscategory"
>
          <h1
                    className="skillsheader"
          >
                     Front End Skills 
          </h1>
          <div
                    className="skilldiv"
          >
                    <img
                              className="skillimage"
                              src="js.png"
                    />
          </div>
          <div
                    className="skilldiv"
          >
                    <img
                              className="skillimage"
                              src="react.png"
                    />
          </div>
          <div
                    className="skilldiv"
          >
                    <img
                              className="skillimage"
                              src="redux.png"
                    />
          </div>
          <div
                    className="skilldiv"
          >
                    <img
                              className="skillimage"
                              src="html5.png"
                    />
          </div>
          <div
                    className="skilldiv"
          >
                    <img
                              className="skillimage"
                              src="css3.png"
                    />
          </div>
          <div
                    className="skilldiv"
          >
                    <img
                              className="skillimage"
                              src="reactrouter.png"
                    />
          </div>
          <div
                    className="skilldiv"
          >
                    <img
                              className="skillimage"
                              src="semantic.png"
                    />
          </div>
          <div
                    className="skilldiv"
          >
                    <img
                              className="skillimage"
                              src="materialize.png"
                    />
          </div>
</div>,
      },
      "_renderedNodeType": 0,
      "_rootNodeID": 0,
      "_topLevelWrapper": null,
      "_updateBatchNumber": null,
      "_warnedAboutRefsInRender": false,
    },
    "getRenderOutput": [Function],
    "render": [Function],
  },
  "root": [Circular],
  "unrendered": <FrontEndSkills />,
}
`;

exports[`Skills OtherSkills renders 1`] = `
ShallowWrapper {
  "complexSelector": ComplexSelector {
    "buildPredicate": [Function],
    "childrenOfNode": [Function],
    "findWhereUnwrapped": [Function],
  },
  "length": 1,
  "node": <div
    className="skillscategory"
>
    <h1
        className="skillsheader"
    >
         Testing & Other Skills 
    </h1>
    <div
        className="skilldiv"
    >
        <img
            className="skillimage"
            src="jest.png"
        />
    </div>
    <div
        className="skilldiv"
    >
        <img
            className="skillimage"
            src="mocha.png"
        />
    </div>
    <div
        className="skilldiv"
    >
        <img
            className="skillimage"
            src="chai.png"
        />
    </div>
    <div
        className="skilldiv"
    >
        <img
            className="skillimage"
            src="supertest.png"
        />
    </div>
    <div
        className="skilldiv"
    >
        <img
            className="skillimage"
            src="git.png"
        />
    </div>
    <div
        className="skilldiv"
    >
        <img
            className="skillimage"
            src="github.png"
        />
    </div>
    <div
        className="skilldiv"
    >
        <img
            className="skillimage"
            src="heroku.png"
        />
    </div>
    <div
        className="skilldiv"
    >
        <img
            className="skillimage"
            src="npm.png"
        />
    </div>
</div>,
  "nodes": Array [
    <div
      className="skillscategory"
>
      <h1
            className="skillsheader"
      >
             Testing & Other Skills 
      </h1>
      <div
            className="skilldiv"
      >
            <img
                  className="skillimage"
                  src="jest.png"
            />
      </div>
      <div
            className="skilldiv"
      >
            <img
                  className="skillimage"
                  src="mocha.png"
            />
      </div>
      <div
            className="skilldiv"
      >
            <img
                  className="skillimage"
                  src="chai.png"
            />
      </div>
      <div
            className="skilldiv"
      >
            <img
                  className="skillimage"
                  src="supertest.png"
            />
      </div>
      <div
            className="skilldiv"
      >
            <img
                  className="skillimage"
                  src="git.png"
            />
      </div>
      <div
            className="skilldiv"
      >
            <img
                  className="skillimage"
                  src="github.png"
            />
      </div>
      <div
            className="skilldiv"
      >
            <img
                  className="skillimage"
                  src="heroku.png"
            />
      </div>
      <div
            className="skilldiv"
      >
            <img
                  className="skillimage"
                  src="npm.png"
            />
      </div>
</div>,
  ],
  "options": Object {},
  "renderer": ReactShallowRenderer {
    "_instance": ShallowComponentWrapper {
      "_calledComponentWillUnmount": false,
      "_compositeType": 2,
      "_context": Object {},
      "_currentElement": <OtherSkills />,
      "_debugID": 7,
      "_hostContainerInfo": null,
      "_hostParent": null,
      "_instance": StatelessComponent {
        "_reactInternalInstance": [Circular],
        "context": Object {},
        "props": Object {},
        "refs": Object {},
        "state": null,
        "updater": Object {
          "enqueueCallback": [Function],
          "enqueueCallbackInternal": [Function],
          "enqueueElementInternal": [Function],
          "enqueueForceUpdate": [Function],
          "enqueueReplaceState": [Function],
          "enqueueSetState": [Function],
          "isMounted": [Function],
          "validateCallback": [Function],
        },
      },
      "_mountOrder": 4,
      "_pendingCallbacks": null,
      "_pendingElement": null,
      "_pendingForceUpdate": false,
      "_pendingReplaceState": false,
      "_pendingStateQueue": null,
      "_renderedComponent": NoopInternalComponent {
        "_currentElement": <div
          className="skillscategory"
>
          <h1
                    className="skillsheader"
          >
                     Testing & Other Skills 
          </h1>
          <div
                    className="skilldiv"
          >
                    <img
                              className="skillimage"
                              src="jest.png"
                    />
          </div>
          <div
                    className="skilldiv"
          >
                    <img
                              className="skillimage"
                              src="mocha.png"
                    />
          </div>
          <div
                    className="skilldiv"
          >
                    <img
                              className="skillimage"
                              src="chai.png"
                    />
          </div>
          <div
                    className="skilldiv"
          >
                    <img
                              className="skillimage"
                              src="supertest.png"
                    />
          </div>
          <div
                    className="skilldiv"
          >
                    <img
                              className="skillimage"
                              src="git.png"
                    />
          </div>
          <div
                    className="skilldiv"
          >
                    <img
                              className="skillimage"
                              src="github.png"
                    />
          </div>
          <div
                    className="skilldiv"
          >
                    <img
                              className="skillimage"
                              src="heroku.png"
                    />
          </div>
          <div
                    className="skilldiv"
          >
                    <img
                              className="skillimage"
                              src="npm.png"
                    />
          </div>
</div>,
        "_debugID": 8,
        "_renderedOutput": <div
          className="skillscategory"
>
          <h1
                    className="skillsheader"
          >
                     Testing & Other Skills 
          </h1>
          <div
                    className="skilldiv"
          >
                    <img
                              className="skillimage"
                              src="jest.png"
                    />
          </div>
          <div
                    className="skilldiv"
          >
                    <img
                              className="skillimage"
                              src="mocha.png"
                    />
          </div>
          <div
                    className="skilldiv"
          >
                    <img
                              className="skillimage"
                              src="chai.png"
                    />
          </div>
          <div
                    className="skilldiv"
          >
                    <img
                              className="skillimage"
                              src="supertest.png"
                    />
          </div>
          <div
                    className="skilldiv"
          >
                    <img
                              className="skillimage"
                              src="git.png"
                    />
          </div>
          <div
                    className="skilldiv"
          >
                    <img
                              className="skillimage"
                              src="github.png"
                    />
          </div>
          <div
                    className="skilldiv"
          >
                    <img
                              className="skillimage"
                              src="heroku.png"
                    />
          </div>
          <div
                    className="skilldiv"
          >
                    <img
                              className="skillimage"
                              src="npm.png"
                    />
          </div>
</div>,
      },
      "_renderedNodeType": 0,
      "_rootNodeID": 0,
      "_topLevelWrapper": null,
      "_updateBatchNumber": null,
      "_warnedAboutRefsInRender": false,
    },
    "getRenderOutput": [Function],
    "render": [Function],
  },
  "root": [Circular],
  "unrendered": <OtherSkills />,
}
`;

exports[`Skills SkillsHero renders 1`] = `
ShallowWrapper {
  "complexSelector": ComplexSelector {
    "buildPredicate": [Function],
    "childrenOfNode": [Function],
    "findWhereUnwrapped": [Function],
  },
  "length": 1,
  "node": <div
    id="skillshero"
>
    <Container>
        <h1
            id="skillsmainheader"
        >
            Skills 
        </h1>
        <Grid
            columns={3}
        >
            <GridRow>
                <GridColumn>
                    <FrontEndSkills />
                </GridColumn>
                <GridColumn>
                    <BackEndSkills />
                </GridColumn>
                <GridColumn>
                    <OtherSkills />
                </GridColumn>
            </GridRow>
        </Grid>
    </Container>
</div>,
  "nodes": Array [
    <div
      id="skillshero"
>
      <Container>
            <h1
                  id="skillsmainheader"
            >
                  Skills 
            </h1>
            <Grid
                  columns={3}
            >
                  <GridRow>
                        <GridColumn>
                              <FrontEndSkills />
                        </GridColumn>
                        <GridColumn>
                              <BackEndSkills />
                        </GridColumn>
                        <GridColumn>
                              <OtherSkills />
                        </GridColumn>
                  </GridRow>
            </Grid>
      </Container>
</div>,
  ],
  "options": Object {},
  "renderer": ReactShallowRenderer {
    "_instance": ShallowComponentWrapper {
      "_calledComponentWillUnmount": false,
      "_compositeType": 2,
      "_context": Object {},
      "_currentElement": <SkillsHero />,
      "_debugID": 1,
      "_hostContainerInfo": null,
      "_hostParent": null,
      "_instance": StatelessComponent {
        "_reactInternalInstance": [Circular],
        "context": Object {},
        "props": Object {},
        "refs": Object {},
        "state": null,
        "updater": Object {
          "enqueueCallback": [Function],
          "enqueueCallbackInternal": [Function],
          "enqueueElementInternal": [Function],
          "enqueueForceUpdate": [Function],
          "enqueueReplaceState": [Function],
          "enqueueSetState": [Function],
          "isMounted": [Function],
          "validateCallback": [Function],
        },
      },
      "_mountOrder": 1,
      "_pendingCallbacks": null,
      "_pendingElement": null,
      "_pendingForceUpdate": false,
      "_pendingReplaceState": false,
      "_pendingStateQueue": null,
      "_renderedComponent": NoopInternalComponent {
        "_currentElement": <div
          id="skillshero"
>
          <Container>
                    <h1
                              id="skillsmainheader"
                    >
                              Skills 
                    </h1>
                    <Grid
                              columns={3}
                    >
                              <GridRow>
                                        <GridColumn>
                                                  <FrontEndSkills />
                                        </GridColumn>
                                        <GridColumn>
                                                  <BackEndSkills />
                                        </GridColumn>
                                        <GridColumn>
                                                  <OtherSkills />
                                        </GridColumn>
                              </GridRow>
                    </Grid>
          </Container>
</div>,
        "_debugID": 2,
        "_renderedOutput": <div
          id="skillshero"
>
          <Container>
                    <h1
                              id="skillsmainheader"
                    >
                              Skills 
                    </h1>
                    <Grid
                              columns={3}
                    >
                              <GridRow>
                                        <GridColumn>
                                                  <FrontEndSkills />
                                        </GridColumn>
                                        <GridColumn>
                                                  <BackEndSkills />
                                        </GridColumn>
                                        <GridColumn>
                                                  <OtherSkills />
                                        </GridColumn>
                              </GridRow>
                    </Grid>
          </Container>
</div>,
      },
      "_renderedNodeType": 0,
      "_rootNodeID": 0,
      "_topLevelWrapper": null,
      "_updateBatchNumber": null,
      "_warnedAboutRefsInRender": false,
    },
    "getRenderOutput": [Function],
    "render": [Function],
  },
  "root": [Circular],
  "unrendered": <SkillsHero />,
}
`;
